game.rb

```diff
@@ -51,8 +50,34 @@ def play(field)
   end
 
   def insert(field)
-    letter = field[0]
-    number = field[1]
+    first = field[0].to_i
+    second = field[1].to_i
+
+    if @board[first][second] == '   '
+      @board[first][second] = @turn == 1 ? ' O ' : ' X '
+      @turn = @turn == 1 ? 2 : 1
+      check_winner
+    else
+      puts 'The field is not empty.'
+    end
+
+    message
+  end
+
+  def check_winner
+
+    combos = [
+      [@board[0][0], @board[0][1], @board[0][2]],
+      [@board[1][0], @board[1][1], @board[1][2]],
+      [@board[2][0], @board[2][1], @board[2][2]],
+      [@board[0][0], @board[1][0], @board[2][0]],
+      [@board[0][1], @board[1][1], @board[2][1]],
+      [@board[0][2], @board[1][2], @board[2][2]],
+      [@board[0][0], @board[1][1], @board[2][2]],
+      [@board[2][0], @board[1][1], @board[0][2]]
+    ]
+
+    @game_over = true if combos.include?([' X ', ' X ', ' X ']) || combos.include?([' O ', ' O ', ' O '])
```

Comment:
Do you need to check specifically for `X` or `O` ? Checking that a row, column or diagonal has `three` of the `same` should be enough.